gpu/utils/include/pcl/gpu/utils/device/algorithm.hpp

```diff
@@ -63,7 +63,7 @@ namespace pcl
                 {
                     first = middle;
                     ++first;
-                    len = len - half - 1;
+                    len -= half - 1;
```

Comment:
This is not equivalent.

Reply:
👍 I checked already all other changes for non-equivalent changes, but not `+=` or `-=`, but I forgot `-=` can change behavior too.